### DISCUSSION TITLE
Alo mí bạn nghe rõ trả lời nhaa

### DISCUSSION THREAD

**Discussion by GiaHan06** [GiaHan06/Lop-10A02-Nhom-02#5] [General]
cái này thầy hong nói khi nào nộp hết á nên mọi người cứ từ từ cũng được nha, để chiều xem thầy có nói gì hong

**Comment by GiaHan06** [comment]
Phần bài làm thì mn biết gì cứ ghi ra hết nhaa, còn có gặp khó khăn hay vấn đề ở câu nào cũng nói luôn nha để tui với mấy bạn còn giúp cho nè !!!

**Comment by GiaHan06** [comment]
Ủa có ai hong dạ:((

**Comment by HuynhLeBaoTran** [comment]
có nè

**Comment by HuynhLeBaoTran** [comment]
mn làm trước nho tui bận tí:33

**Reply by GiaHan06** [reply]
> mn làm trước nho tui bận tí:33

oke bà <3

**Comment by GiaHan06** [comment]
mn đừng im lặng như thế chứ:(

**Comment by LTGHan** [comment]
Hi

**Comment by LTGHan** [comment]
Câu 4 lm sao :)))

**Reply by GiaHan06** [reply]
bà xem sgk/ 36,37 đồ á có cách mấy bài mẫu á, hong ấy thì tui làm xong của tui ròi tui xem thử choo

**Comment by GiaHan06** [comment]
@LTGHan ủa câu 3 hong biết có cần giải thích cái 1 luôn hong bà

**Reply by LTGHan** [reply]
Tui cũng định
Nma để sau ik

**Reply by GiaHan06** [reply]
ủa mà câu 4 là sao ta?? hong phải là nó diễn tả ròi hả tròi

**Reply by LTGHan** [reply]
:)))) bởi z tui mới hong biết

**Reply by LTGHan** [reply]
Để đổi avt Xiao bối rối cái

**Reply by LTGHan** [reply]
Tada

**Comment by LTGHan** [comment]
Ủa z có mấy cách diễn tả thuật toán? Tui đọc sách là 2 ó

**Reply by LTGHan** [reply]
Liệt kê và Sơ đồ khối

**Reply by GiaHan06** [reply]
ủa khoann dừng khoảng chừng là 2s
ủa diễn tả với mô tả giống nhau hong?? sao câu 4 thầy kiu diễn tả mà câu 6 thầy kiu mô tả zãy

**Reply by LTGHan** [reply]
Tui hiểu rồi :)))

**Reply by LTGHan** [reply]
À hoy hết hiểu

**Reply by GiaHan06** [reply]
cái nèo hong hiểu

**Reply by LTGHan** [reply]
Cái diễn tả thuật toán từ ví dụ 3 í

**Reply by GiaHan06** [reply]
Đắp mền đi ngụ đii

**Comment by GiaHan06** [comment]
@LTGHan ơi hình như là dùng sơ đồ khối á bà

**Reply by LTGHan** [reply]
:00000000

**Reply by LTGHan** [reply]
Howw

**Reply by GiaHan06** [reply]
tui thấy cũng giống liệt kê nhưng mà mình làm thành sơ đồ thoi á, chắc vậy

**Reply by LTGHan** [reply]
:)))))))

**Comment by GiaHan06** [comment]
@HuynhLeBaoTran , @chaudambaotram , @NguyenNgocQuynhAnh  khi nào mấy bạn làm thì nhắn lên đây cho tui nha để tui vô giúp mn với lại cho mn đỡ bị cu đơn nè. Mà đừng có kiu tui lúc 10h 11h... gì nhe tại lúc đó tui ngủ mất tiêu ròi :(

**Reply by GiaHan06** [reply]
mấy câu nào làm được thì mấy bà làm trước nha, cái nào hong được thì để dành đó rùi đi ngụ đii

**Reply by NguyenNgocQuynhAnh** [reply]
mới làm xong câu 1 nhưng lừi quá nên mai làm hjhj

**Comment by LTGHan** [comment]
How to do vế 2 của câu 4 zzzzzzzzzzzzz

**Reply by GiaHan06** [reply]
hỏi thầy thử đii :3

**Reply by GiaHan06** [reply]
hỏi hum :))

**Reply by LTGHan** [reply]
Hỏi ik bà :))

**Reply by GiaHan06** [reply]
nên hong chòi

**Reply by LTGHan** [reply]
:'))) chứ h sao

**Reply by GiaHan06** [reply]
có khi câu này rất đơn giản nhưng we make it complicated, nên mình cứ go to bed and sleep đi :))

**Reply by LTGHan** [reply]
Z viết lại thôi ha :)))) tại cái đề là dấu chấm còn diễn tả thuật toán là chấm phẩy nên chỉnh lại thui gòi viết y chang ha :)))))

**Comment by NguyenNgocQuynhAnh** [comment]
ai làm câu 7 á thì làm i để tui nhìn vô rồi làm câu 8 nhóeeeeeee

**Reply by GiaHan06** [reply]
tui nói sơ qua câu 7 cho nè
dùng tay thì cầm quả bóng đầu tiên lên, lấy cái cục bóng đó làm đơn vị chuẩn( kiểu cái cục đó là khối lượng lớn nhất nha), rồi cầm từng cục tiếp theo để so sánh, cục nào nặng hơn thì lấy cái cục nặng hơn làm chuẩn tiếp rồi cứ tiếp tục cho đến cục cuối cùng là tìm ra được nha. mà tui đọc trong sách thấy cái "i" là cái gì á tui cũng chưa hiểu nữa nhma nó là zạy á

**Reply by GiaHan06** [reply]
bà hiểu hôn???

**Reply by NguyenNgocQuynhAnh** [reply]
z tui trình bày câu 7 bằng các bước hả

**Reply by NguyenNgocQuynhAnh** [reply]
à nhầm hjhj

**Reply by GiaHan06** [reply]
z zãy chòi bà làm câu 8 hoii mà

**Reply by NguyenNgocQuynhAnh** [reply]
cái ý tưởng á thì tui nên ghi là so sánh khối lượng của quả bóng đang xét với khóa k hay quả bóng thứ nhất á

**Reply by GiaHan06** [reply]
hả là sao bà, tui chưa hiểu ý bà lắm á, khóa k là gì á bà???

**Comment by HuynhLeBaoTran** [comment]
huhu có ai chỉ tui tìm cái input output của câu 7 dớiii

**Reply by GiaHan06** [reply]
input thì tui phân vân là 5 quả bóng với lại khối lượng của từng quả bóng á tại vì là đo bằng tay nên tui hong chắc lắm :(
output: quả bóng có khối lượng lớn nhất

**Comment by chaudambaotram** [comment]
mọi người ơi,mình lam xong câu 7 và câu 8 rồi ấy.Nhưng mình làm theo cách mình hiểu ấy.Mọi người thấy mình lam có sai sót gì không?Nếu có mọi người chỉ giúp mình để mình sửa lại nha.

**Reply by GiaHan06** [reply]
oke bà nè
